src/test/java/co/rsk/bitcoinj/script/RedeemScriptUtils.java

```diff
@@ -8,13 +8,16 @@
 import java.util.Arrays;
 import java.util.List;
 
-public class RedeemScriptUtils {
+public final class RedeemScriptUtils {
 
-    protected static Script createStandardRedeemScript(List<BtcECKey> publicKeys) {
+    private RedeemScriptUtils() {
+    }
+
+    public static Script createStandardRedeemScript(List<BtcECKey> publicKeys) {
         return ScriptBuilder.createRedeemScript(publicKeys.size() / 2 + 1, publicKeys);
     }
 
-    protected static Script createFastBridgeRedeemScript(
+    public static Script createFastBridgeRedeemScript(
```

Comment:
Since these methods are in a `Utils` `class`, which is declared `final`. When you have a `final` `class` and `protected` methods, these methods are marked with weak visibility.